src/lookups.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  config as bskConfig,
+  validateProofs,
+  resolveZoneFileToProfile,
+} from "blockstack";
+import { validateStacksAddress } from "@stacks/transactions";
+import fetch from "node-fetch";
+//@ts-ignore
+import logger from "winston";
+import * as cheerio from "cheerio";
+
+export async function isSubdomainRegistered(fullyQualifiedAddress: string) {
+  try {
+    const nameInfoUrl =
+      bskConfig.network.blockstackAPIUrl + "/v1/names/" + fullyQualifiedAddress;
+    const nameInfoRequest = await fetch(nameInfoUrl, {});
+    const { status } = nameInfoRequest;
+    const nameInfo = await nameInfoRequest.json();
+    if (status == 200) {
+      return nameInfo.status === "registered_subdomain";
+    } else {
+      return false;
+    }
+  } catch (err) {
+    //@ts-ignore
```

Comment:
// ts-ignore should not be used here, there is an option in the tsconfig to not set errors as unknown type, better to use this way I think